File: src/JavaScript/Where-my-anagrams-at.js
function anagrams(word, words) {
  rearrange = string => string.split('').sort().join('');
  return words.filter(w => rearrange(w) === rearrange(word));
}

console.log(anagrams("abba", ["aabb", "abcd", "bbaa", "dada"]));

File: src/Julia/Where-my-anagrams-at.jl


rearrange(str::String)::String = collect(str) |> sort |> join


"""
words 中与 word 拥有相同字母的元素
"""
function anagrams(word::String, words::Vector{String})::Vector{String}
    filter(w -> rearrange(w) == rearrange(word), words)
end


anagrams("abba", ["aabb", "abcd", "bbaa", "dada"])